scholarx/archive/index.html

```diff
@@ -81,15 +88,12 @@
       <div class="col-3 mt-5">
         <h6>FILTER</h6>
         <hr>
-        <div class="dropdown mt-2 mb-5">
-            <button class="btn btn-secondary dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
-                Mentees
-            </button>
-            <ul class="dropdown-menu">
-              <li><a class="dropdown-item" href="#">Action</a></li>
-              <li><a class="dropdown-item" href="#">Another action</a></li>
-              <li><a class="dropdown-item" href="#">Something else here</a></li>
-            </ul>
+        <!--mentor mentee selection-->
+        <div class="mt-2 mb-5">
+            <select id="selection" class="btn btn-secondary dropdown-toggle" >
+                <option value="Mentors">Mentors</option>
```

Comment:
let's use lowercase values
```suggestion
                <option value="mentors">Mentors</option>
```